docs/en/academic/custom-templates.md

```diff
@@ -1,83 +1,134 @@
 
-### Metadata Variables and Syntax
+Variables will be replaced globally. This means that, for example, if the variable `\$PDF_AUTHOR$` is found multiple times in the template, it will be replaced each time.
 
-Since Pandoc accepts YAML frontmatters and allows you to set further options using defaults files, there are many more variables which you can use in your templates, including `if`-statements allowing to check for certain conditions. There are many pre-defined variables that you can use in the built-in templates, but if you define your own templates, you can also define custom ones.
+Zettlr then saves the revised LaTeX template to a temporary directory.
 
-Metadata variables must be explicitly set to use them — either in YAML frontmatters or in defaults files. For example, you have to name authors in your YAML frontmatter in order to have the `$author$` variable available. You can check if the `$author$` variable is set by using the pattern `$if(author)$`, `$else$`, and `$endif$`. A few metadata variables will be set even if you did not explicitly define them, for example the `$title$`-variable, since that one is required by many formats.
+### 4. Prepare the PDF Export
 
-> Note that you can also make use of arbitrary metadata variables in your templates. For example, you could introduce a variable to toggle if a certain HTML-element should be used, or not, and then place this variable in the YAML frontmatters to control that element.
+With the source and template files prepared, Zettlr will prepopulate the Pandoc command variables. One of these commands tells Pandoc that the revised source file from step 3 is your input file. Another command adds the revised LaTeX template; if a custom template is not provided, Zettlr will use its [default template](https://github.com/Zettlr/Zettlr/blob/master/source/main/assets/export.tex). To maximise compatibility, this template is a simplified version of the [Pandoc default template](https://github.com/jgm/pandoc/blob/master/data/templates/default.latex).
 
-Pandoc's templating engine is pretty powerful, so we can not fully dive into the templating syntax. The [Pandoc documentation](https://pandoc.org/MANUAL.html#templates) has a long section on that, including all supported variables for the built-in templates.
+> Many of the commands in the default Pandoc template require additional LaTeX packages. To not confuse users who only need the basics, Zettlr's default template strives for maximum compatibility, not perfect PDFs.
 
-### Templates versus "Reference Docs"
+<!-- TO CHECK: Regarding the next line, I was not able to find the 'table of contents' option in the project propertie; it seems to be much simpler and only asks about what format to export the project in and also about what files to ignore via a glob pattern (there is also a lot of little icons visible at the botton of this project properties menu that seem to not do anything...).-->
```

Comment:
ToC settings on a per-project basis are gone, these can now be adjusted in the defaults files (assets manager) globally for everything.